### PR TITLE
New package: noip2-2.1.9_1

### PR DESCRIPTION
**Pull request by ayghor**
No-ip Dynamic DNS Update Client

### CODE REVIEW

File: srcpkgs/noip2/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+wrksrc="noip-2.1.9-1"
+makedepends=""
+conf_files="/etc/no-ip2.conf"
+short_desc="No-ip Dynamic DNS Update Client"
+maintainer="rogi <rogi@skylittlesystem.org>"
+license="GPL-2"
+checksum=82b9bafab96a0c53b21aaef688bf70b3572e26217b5e2072bdb09da3c4a6f593
+
+do_build() {
+	make CC=$CC
```
Comment (xtraeme) [inline srcpkgs/noip2/template]: should also pass CFLAGS.

File: srcpkgs/noip2/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+do_install() {
+	vinstall noip2 755 usr/bin
+}
+
+post_install() {
```
Comment (xtraeme) [inline srcpkgs/noip2/template]: There's no need to use `post_install` if you are using `do_install` already, move that to  `do_install`.

File: srcpkgs/noip2/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'noip2'
+pkgname=noip2
+version=2.1.9_1
+revision=1
+homepage="http://www.noip.com"
+distfiles="https://www.noip.com/client/linux/noip-duc-linux.tar.gz"
+wrksrc="noip-2.1.9-1"
+makedepends=""
```
Comment (xtraeme) [inline srcpkgs/noip2/template]: Remove empty var.

File: srcpkgs/noip2/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'noip2'
+pkgname=noip2
+version=2.1.9_1
+revision=1
+homepage="http://www.noip.com"
+distfiles="https://www.noip.com/client/linux/noip-duc-linux.tar.gz"
```
Comment (xtraeme) [inline srcpkgs/noip2/template]: Unversioned tarball... this will cause troubles if the tarball is modified upstream.

Comment (ayghor) [inline srcpkgs/noip2/template]: yea, but that is something I can't change

Comment (ayghor) [inline srcpkgs/noip2/template]: couldn't find a link to exact version

Comment (xtraeme) [inline srcpkgs/noip2/template]: No problem. Keep it as is for now.

**Comment by ayghor** [comment]
Fixed stuff you said!

File: srcpkgs/noip2/template [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'noip2'
+pkgname=noip2
+version=2.1.9_1
+revision=2
```
Comment (xtraeme) [inline srcpkgs/noip2/template]: Please set revision to 1 for new packages.

File: srcpkgs/noip2/template [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+maintainer="rogi <rogi@skylittlesystem.org>"
+license="GPL-2"
+checksum=82b9bafab96a0c53b21aaef688bf70b3572e26217b5e2072bdb09da3c4a6f593
+
+do_build() {
+	make CC=$CC CFLAGS="$CFLAGS"
+}
+
+do_install() {
+	vbin noip2 755 usr/bin
```
Comment (xtraeme) [inline srcpkgs/noip2/template]: This vbin line is wrong, use `vbin noip2`.

Comment (ayghor) [inline srcpkgs/noip2/template]: omg sry

Comment (ayghor) [inline srcpkgs/noip2/template]: note xlint didn't catch this one

**Pull request merged**